_docs/manage/policies.md

```diff
@@ -0,0 +1,118 @@
+```
+
+These files are provided using the optional argument `--policies`
+```bash
+$ storageos policy create --policies='p1.jsonl,p2.jsonl'
+
+$ storageos policy ls
+ID                                     USER                GROUP               READONLY            APIGROUP            RESOURCE            NAMESPACE           NONRESOURCEPATH
+24e90cfe-9dbe-482c-3b51-8d2a9c0e57fd   foo                                     false                                                       *
+34c868ef-4e4b-fb71-f0f7-fa94e4652776   bar                                     false                                                       testing
+8ced2820-7ad1-bd46-4d9f-d7dab1ba4c67                       baz                 false                                                       restricted
+```
+#### Adding using stdin
+The final method of adding policies is using the stdin for the `storageos policy create` command, using the `--stdin` option. The stdin accepts input in both json form and jsonl form.
+
+json form:
+```bash
+$ echo '{"spec":{"user":"foo", "namespace": "*"}}' | storageos policy create --stdin
+
+$ storageos policy ls
+ID                                     USER                GROUP               READONLY            APIGROUP            RESOURCE            NAMESPACE           NONRESOURCEPATH
+1dcd73d1-23de-0329-7501-fc7aa60affb2   foo                                     false                                                       *
+```
+
+jsonl form:
+```bash
+$ cat p1.jsonl | storageos policy create --stdin
+
+$ storageos policy ls
+ID                                     USER                GROUP               READONLY            APIGROUP            RESOURCE            NAMESPACE           NONRESOURCEPATH
+976a74bd-630c-eeb5-8c33-b033b99dcf68   foo                                     false                                                       *
+f1e33492-aa9b-3922-7824-383941d37a5d   bar                                     false                                                       testing
+```
+
+### Listing current policies
+Policies live in the system can be displayed using the `storageos policy ls` command.
+
+```bash
+$ storageos policy ls
+ID                                     USER                GROUP               READONLY            APIGROUP            RESOURCE            NAMESPACE           NONRESOURCEPATH
+99b04c50-32a7-4869-8e23-1e6b8b887228   foo                                     false                                                       *
+c14aa3ff-99f7-d0fb-41e9-ad45b0919851   bar                                     false                                                       testing
+13de44a2-c50a-c225-9f65-871cfad0c838                       baz                 false                                                       restricted
+```
+
+### Removing policies
+Policies can be removed from the system using the `storageos policy rm` command. Multiple policies can be selected at once.
```

Comment:
@oicheryl I'm not sure what you mean by instantly. By the time the command has returned they have been deleted from the system, however the delete is not atomic. Requests can come in while policies are being deleted, and potentially be checked against some subset of the policies.